wire-ios-data-model/Resources/Databases Changelog.md

```diff
@@ -4,6 +4,11 @@ As it is hard to spot changes from version to version of database models (.xcdat
 
 ## zmessaging
 
+### 2.122.0
+
+* Remove `ToDeleted` entity from 2.120.0
+* Cleanup `needsToUploadSignalingKeys`, `apsVerificationKey`, `apsDecryptionKey` from `UserClient`
```

Comment:
"APS signaling keys were used in the past to encrypt/decrypt the content of push notification back in the day when messages weren't end-2-end encrypted."